root/components/EntityTabLink.js

```diff
@@ -13,13 +13,22 @@ import EntityLink from '../static/scripts/common/components/EntityLink';
 
 type Props = {
   +content: string,
+  +disabled?: boolean,
   +entity: CoreEntityT | CollectionT,
   +selected: boolean,
   +subPath: string,
 };
 
-const EntityTabLink = ({selected, ...linkProps}: Props) => (
-  <li className={selected ? 'sel' : null}>
+const EntityTabLink = ({disabled, selected, ...linkProps}: Props) => (
+  <li
+    className={
+      selected || disabled
+        ? (selected ? 'sel' : '') +
+          (selected && disabled ? ' ' : '') +
+          (disabled ? 'disabled' : '')
+        : null
```

Comment:
Since a tab can't be both selected and disabled (right?), would this be equivalent?
```JS
(selected ? 'sel' : (disabled ? 'disabled' : null))
```

Reply:
It can, actually - but just by manually going to /discids. I didn't block that because there might be a case where (due to a bug or whatnot, I think some bugs related to merges still exist currently) a release that shouldn't have discIDs ends up having some.